### ISSUE TITLE
What is sc88proe.bin?

### ISSUE THREAD

**Issue by ghost** [ipwndev/fishshell2#1]
The title says it all. I know the file contains sample data for MIDI file playback, like a soundfont, but what format is in? What is the ".bin" format?

**Comment by ipwndev** [comment]
My first guess was that it was an ARM9 binary(false) would be that it is a binary, due to the .bin extension.  Just probing around with a hex editor, the first line has references to "PFS1".  My understanding is that it may be referring to FastTracker 1, (perhaps).  After bit more analysis(and a healthy amount of guessing), I have found there is a header containing some data I have yet to identify, but after that, it is  a few hundred samples of 16bit PCM audio(each around a second long).  Id imagine you could go and change certain segments in Audacity to those you want.  As long as the preamble(first few seconds, think of it as an index) at the beginning remains the same, the notes will also stay the same(untested hypothesis). 

To open "sc88proe.bin" in Audacity, create a new project, go to import and select "Import Raw Data" and choose the "sc88proe.bin" file from wherever its stored.  A menu to choose options will come up, you will want to change the encoding to "Signed 16-bit PCM" and the byte order to little-endian.  Also the start offset is at zero bytes, and you will import 100% of the file to import. Regarding how many audio channels, 1 Channel "Mono" audio worked baest.  from here you can listen to the samples and replace individual samples.  

Based on the question, I am assuming you wish to know what the format is so you can convert another sound font, to use on the Nintendo DS/moonshell?  If there is interest and If I can decode the header, I might be able to come up with an automated solution.

Also, just a note I am not the author for pretty much all of this project(just trying to get it working under DSi Mode),  I will add their credits into the readme when I push the next commit.

**Comment by ghost** [comment]
Yes, I would like it if you could decode the header and figure out whether it could be converted
